src/components/common/PaginatedTxns/index.tsx

```diff
@@ -20,20 +20,18 @@ const PaginatedTxns = ({ useTxns }: { useTxns: typeof useTxHistory | typeof useT
 
   useEffect(() => {
     if (page?.results.length) {
-      setAllResults((prev) => {
-        const newResults = prev.concat(page.results)
-
-        if (isQueue) {
-          trackEvent({
-            ...TX_LIST_EVENTS.QUEUED_TXS,
-            label: newResults.length,
-          })
-        }
+      setAllResults((prev) => prev.concat(page.results))
+    }
+  }, [page])
 
-        return newResults
+  useEffect(() => {
+    if (isQueue && allResults.length > 0) {
```

Comment:
We don't want to track 0? Was it like this in safe-react?

Reply:
Yes. Do you have thoughts @johannesmoormann?

Reply:
After discussing it, we'll track all amounts.